runtime/android/core_internal/src/org/xwalk/core/internal/XWalkDownloadListenerImplInternal.java

```diff
@@ -23,10 +23,11 @@
 import java.io.OutputStream;
 
 import org.xwalk.core.internal.AndroidProtocolHandler;
-import org.xwalk.core.internal.DownloadListener;
 import org.xwalk.core.internal.R;
+import org.xwalk.core.internal.XWalkDownloadListenerInternal;
 
-class XWalkDownloadListenerImpl implements DownloadListener {
+@XWalkAPI(impl = XWalkDownloadListenerInternal.class, createExternally = true)
```

Comment:
We don't need to expose XWalkDownloadListenerImpl. But this class could be the default behavior if the developer doesn't override XWalkDownloadListener